app/src/test/kotlin/com/example/cleanarchitecturesolution/features/episode/data/repository/EpisodeRepositoryTest.kt

```diff
@@ -22,6 +24,13 @@ import org.junit.jupiter.api.extension.ExtendWith
 @ExtendWith(MainDispatcherRule::class)
 internal class EpisodeRepositoryTest {
 
+    private lateinit var errorWrapper: ErrorWrapper
```

Comment:
Możesz tutaj nieco zmniejszyć ilość kodu stosując po prostu zapis
```
private val errorWrapper = mockk<ErrorWrapper>(relaxed = true)
```
Wtedy tę metodę `before` będzie można usunąć. Mock utworzy się niezależnie dla każdego testu. Możesz to zrobić analogicznie w innych klasach, gdzie stosujesz ten sam zapis.